src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheck.java

```diff
@@ -255,11 +266,14 @@ private static boolean isInNoBraceControlStatement(DetailAST ast) {
     }
 
     /**
-     * Tests whether the given AST is used in the "assignment in while" idiom.
+     * Tests whether the given AST is used in the "assignment in loop" idiom.
      * <pre>
      * String line;
      * while ((line = bufferedReader.readLine()) != null) {
-     *    // process the line
+     *     // process the line
+     * }
+     * for (;(value = reader.read()) != 0;) {
+     *     // process
```

Comment:
please make indentation here 2 as well, like in javadoc for class

Reply:
done

Reply:
not changed. Please also see line 273 for same issue

Reply:
done. please review the update